pages/shared/reference/materials/pbrshader.md

```diff
@@ -0,0 +1,5 @@
+---
+#title: The PBR Shader
+---
+This page documents all the shader parameters of the PBR shader in all Strata games.
+(TODO: Document this!)
```

Comment:
I've got a little shader documentation markdown generator that could be used as the basis for this page.

https://github.com/JJL772/strata-shader-docs/releases/download/v2024.1.1-0/shaders.zip

Reply:
Basically ported the contents of the generated page underneath my text. The VDC splits parameters into their own subheadings for readability (see [LightmappedGeneric](https://developer.valvesoftware.com/wiki/LightmappedGeneric)), so I may do that later. Its usable atm, however.